tests/openwisp2/firmware/fake-img.bin

```diff
@@ -0,0 +1 @@
+dummy
```

Comment:
haha, can you believe the cause of the test failures is this line?? :joy: :joy:

Reply:
OMG :man_facepalming: this is somewhat required otherwise uploading an empty firmware image will fail

Reply:
why will it fail? These tests were passing fine before.

Reply:
because you cannot upload an empty file

Reply:
@PabloCastellano I'd create a new file and leave that one alone.

Reply:
And I'd add a comment in the new file explaining it's needed to test the API and it cannot be empty.